generators/app/templates/.gitignore

```diff
@@ -1,35 +1,49 @@
-################################################
-# macOS
-#################################################
-*.DS_Store
-.AppleDouble
-.LSOverride
+# System
+# ---------------------------
+.DS_Store
+Thumbs.db
 
-# Icon must end with two \r
-Icon
+# Config
+# ---------------------------
+*.esproj
+*.sublime-project
+*.sublime-workspace
+*.tmproj
+*.tmproject
+.env
+.env.php
+.idea
+.project
+.settings
 
-# Thumbnails
-._*
+# Compiled Files
+# ---------------------------
+/dist/
 
-# Files that might appear in the root of a volume
-.DocumentRevisions-V100
-.fseventsd
-.Spotlight-V100
-.TemporaryItems
-.Trashes
-.VolumeIcon.icns
-.com.apple.timemachine.donotpresent
+# Craft
+# ---------------------------
+/craft/plugins/*
+/craft/rev-manifest.json
```

Comment:
You mean you're not a fan of storing it in the craft folder? If so, I agree. It's not specifically tied to Craft enough to be tied to that folder.